### PR TITLE
Parameter typo fix for New-AZAutomationUpdateManagementAzureQuery

### PR DESCRIPTION
**Pull request by jemex**
## Checklist

- [ x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x ] The title of the PR is clear and informative
- [x ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/Automation/Automation.Test/ScenarioTests/UpdateDynamicGroupPrePostTests.ps1 [hunk 1]
```diff
@@ -295,6 +295,62 @@ Test-CreateAndGetSoftwareUpdateConfigurationWithAzureDynamicGroupsOnlyWithOutTag
                                        -StartTime $startTime `
                                        -ForUpdate
 
+$query1Scope = @(
+        "/subscriptions/cd45f23b-b832-4fa4-a434-1bf7e6f14a5a/resourceGroups/mms-wcus"
```
Comment (markcowl) [inline src/Automation/Automation.Test/ScenarioTests/UpdateDynamicGroupPrePostTests.ps1]: This hard-coded subscription Id will cause problems in re-recording the test

File: src/Automation/Automation.Test/ScenarioTests/UpdateDynamicGroupPrePostTests.ps1 [hunk 2]
```diff
@@ -295,6 +295,62 @@ Test-CreateAndGetSoftwareUpdateConfigurationWithAzureDynamicGroupsOnlyWithOutTag
...
+    )
+
+    $query1Location =@("Japan East", "UK South")
```
Comment (markcowl) [inline src/Automation/Automation.Test/ScenarioTests/UpdateDynamicGroupPrePostTests.ps1]: Shoudl not hard code locations , but use resource manager APIs to find available locations, and select from that list (you cna prefer a location, if it exists)

File: src/Automation/Automation/ChangeLog.md [hunk 3]
```diff
@@ -18,7 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
-
+ * Bug fix for NewAZAutomationUpdateManagementAzureQuery
 ## Version 1.2.1
```
Comment (markcowl) [inline src/Automation/Automation/ChangeLog.md]: Need a line of space before the next heading

File: src/Automation/Automation/Cmdlet/UpdateManagement/NewAutomationUpdateManagementAzureQuery .cs [hunk 4]
```diff
@@ -37,6 +37,9 @@ public class NewAutomationUpdateManagementAzureQuery : AzureAutomationBaseCmdlet
         [Parameter(Mandatory = false, HelpMessage = "List of locations for azure virtual machines.")]
         public string[] Locaton { get; set; }
 
+        [Parameter(Mandatory = false, HelpMessage = "List of locations for azure virtual machines.")]
+        public string[] Location { get; set; }
```
Comment (markcowl) [inline src/Automation/Automation/Cmdlet/UpdateManagement/NewAutomationUpdateManagementAzureQuery .cs]: Shopuld use a LocationCompleter attribute for this

Comment (jemex) [reply src/Automation/Automation/Cmdlet/UpdateManagement/NewAutomationUpdateManagementAzureQuery .cs]: Hi @markcowl , can you explain little bit about this? is there existing attribute with the Name "LocationCompleter" or should I create one? Is there example of this that has been used in other code?

Comment (markcowl) [reply src/Automation/Automation/Cmdlet/UpdateManagement/NewAutomationUpdateManagementAzureQuery .cs]: See: https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/design-guidelines/parameter-best-practices.md#location-completer

Comment (jemex) [reply src/Automation/Automation/Cmdlet/UpdateManagement/NewAutomationUpdateManagementAzureQuery .cs]: @markcowl , Thank you so much. Updated

File: src/Automation/Automation.Test/ScenarioTests/UpdateDynamicGroupPrePostTests.ps1 (hunk 2 above)
Comment (jemex) [reply src/Automation/Automation.Test/ScenarioTests/UpdateDynamicGroupPrePostTests.ps1]: @markcowl , Thank you for the feedback. Can you give me any doc location for the API usage that help to find the available locations? I couldn't easily locate the API. any hint will be appreciated

File: src/Automation/Automation.Test/ScenarioTests/UpdateDynamicGroupPrePostTests.ps1 (hunk 1 above)
Comment (jemex) [reply src/Automation/Automation.Test/ScenarioTests/UpdateDynamicGroupPrePostTests.ps1]: these tests need prep environements before they run. in order to re-record the test, the steps are documented on the top of the file.

**Comment by MiYanni** [comment]
@jemex It looks like your commit history is messed up. You shouldn't have 142 commits and 329 file changes. I'll pull the latest from the `master` branch. If that doesn't fix it, I'd make a new branch from `master` and make you changes in that branch, close this PR, and open a new one from that branch.

**Comment by jemex** [comment]
Closing this PR as commit history is badly a conflicted.

**Pull request closed without merging**